sensu/config.go

```diff
@@ -61,9 +76,10 @@ func NewConfigFromFlagSet(flagset *configFlagSet) (*Config, error) {
 		if err := json.Unmarshal(buf, &cfg.config); err != nil {
 			return nil, err
 		}
+		return &cfg, nil
 	}
+	return nil, errors.New("No config file is set")
```

Comment:
I think we don't want to return an error here, because the client is able to subscribe to checks from the Sensu server via the `SENSU_CLIENT_SUBSCRIPTIONS` environment variable and you can use `RABBITMQ_URI` to specify the URI of the Sensu server, which I believe lets you run the client without a config file.